packages/kubernetes-1.29/kubernetes-1.29.spec

```
@@ -190,6 +219,7 @@ install -p -m 0644 %{S:15} %{buildroot}%{_cross_datadir}/logdog.d
 %dir %{_cross_libexecdir}/kubernetes
 %{_cross_libexecdir}/kubernetes/kubelet-plugins
 %{_cross_datadir}/logdog.d/logdog.kubelet.conf
+%{_cross_datadir}/eks/amazon-eks-pause.tar
```

since it's an arch-specific binary rather than human-readable:
```suggestion
%{_cross_libexecdir}/kubernetes/kubernetes-pause.tar
```